diff -ruN flopsync/drivers/nrf24l01.cpp flopsync_temp/drivers/nrf24l01.cpp
--- flopsync/drivers/nrf24l01.cpp	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/drivers/nrf24l01.cpp	2013-07-28 23:11:15.000000000 +0200
@@ -92,15 +92,15 @@
         delayUs(1);
     }
     
-    if(this->mode==SLEEP)
-    {
-        // Wait 1.5ms if old mode was SLEEP
-        // Note that the time to send the SPI commands is around 100us resulting
-        // in this code taking around 1.6ms
-        AuxiliaryTimer& timer=AuxiliaryTimer::instance();
-        timer.initTimeoutTimer(static_cast<int>(0.0015f*16384+0.5f));
-        timer.waitForTimeout();
-    }
+//     if(this->mode==SLEEP)
+//     {
+//         // Wait 1.5ms if old mode was SLEEP
+//         // Note that the time to send the SPI commands is around 100us resulting
+//         // in this code taking around 1.6ms
+//         AuxiliaryTimer& timer=AuxiliaryTimer::instance();
+//         timer.initTimeoutTimer(static_cast<int>(0.0015f*16384+0.5f));
+//         timer.waitForTimeout();
+//     }
     
 //    if(mode==RX) nrf::ce::high(); //Start receiving right now if RX
     this->mode=mode;
diff -ruN flopsync/drivers/rtc.cpp flopsync_temp/drivers/rtc.cpp
--- flopsync/drivers/rtc.cpp	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/drivers/rtc.cpp	2013-07-28 23:03:41.000000000 +0200
@@ -150,9 +150,16 @@
     TIM7->CR1 |= TIM_CR1_CEN;
 }
 
+//--------------
+typedef Gpio<GPIOC_BASE,8> fixme;
+//--------------
+
 bool AuxiliaryTimer::waitForPacketOrTimeout()
 {
     FastInterruptDisableLock dLock;
+    
+    fixme::low();
+   
     while(packetReceived==false && timeout==false)
     {
         timeoutWaiting=Thread::IRQgetCurrentThread();
@@ -163,12 +170,18 @@
         }
     }
     packetReceived=false;
+    
+    fixme::high();   
+    
     return timeout;
 }
 
 void AuxiliaryTimer::waitForTimeout()
 {
     FastInterruptDisableLock dLock;
+    
+    fixme::low();
+   
     while(timeout==false)
     {
         timeoutWaiting=Thread::IRQgetCurrentThread();
@@ -178,6 +191,9 @@
             Thread::yield();
         }
     }
+    
+    fixme::high();
+   
 }
 
 void AuxiliaryTimer::stopTimeoutTimer()
@@ -246,6 +262,9 @@
 void Rtc::wait()
 {
     FastInterruptDisableLock dLock;
+    
+    fixme::low();
+   
     while(rtcInterrupt==false)
     {
         waiting=Thread::IRQgetCurrentThread();
@@ -255,6 +274,9 @@
             Thread::yield();
         }
     }
+    
+    fixme::high();
+   
 }
 
 void Rtc::sleepAndWait()
@@ -264,7 +286,9 @@
     {
         FastInterruptDisableLock dLock;
         if(rtcInterrupt) return;
-
+        
+        fixme::low();
+       
         sleepAgain:
         PWR->CR |= PWR_CR_LPDS;
         SCB->SCR |= SCB_SCR_SLEEPDEEP_Msk;
@@ -315,6 +339,9 @@
 
         RCC->CR |= RCC_CR_PLLON;
         while((RCC->CR & RCC_CR_PLLRDY)==0) ;
+        
+        fixme::high();
+       
         RCC->CFGR &= ~RCC_CFGR_SW;
         RCC->CFGR |= RCC_CFGR_SW_PLL;    
         while((RCC->CFGR & RCC_CFGR_SWS)!=0x08) ; 
diff -ruN flopsync/experiment.conf flopsync_temp/experiment.conf
--- flopsync/experiment.conf	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/experiment.conf	2013-07-28 23:47:29.000000000 +0200
@@ -5,7 +5,7 @@
 # General config parameters
 experiment_name="again10"
 experiment_time="30*60"
-sync_period="15"
+sync_period="5"
 relative_clock="0"
 interactive_rootnode="0"
 event_timestamping="0"
@@ -20,4 +20,4 @@
 
 # Node 2
 node2_file="flopsync_dynamic_comb.cpp flopsync2.cpp"
-node2_second_hop="1"
+node2_second_hop="0"
diff -ruN flopsync/flopsync2.cpp flopsync_temp/flopsync2.cpp
--- flopsync/flopsync2.cpp	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/flopsync2.cpp	2013-07-28 23:41:42.000000000 +0200
@@ -138,9 +138,9 @@
     }
     short clockCorrection=r.first;
     receiverWindow=r.second;
-    
-    printf("e=%d u=%d w=%d%s\n",measuredFrameStart-computedFrameStart,
-        clockCorrection,receiverWindow,timeout ? " (miss)" : "");
+
+//     printf("e=%d u=%d w=%d%s\n",measuredFrameStart-computedFrameStart,
+//         clockCorrection,receiverWindow,timeout ? " (miss)" : "");
     
     //Correct frame start considering hops
     measuredFrameStart-=hop*retransmitDelta;
diff -ruN flopsync/flopsync2.h flopsync_temp/flopsync2.h
--- flopsync/flopsync2.h	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/flopsync2.h	2013-07-28 23:47:15.000000000 +0200
@@ -228,7 +228,7 @@
     unsigned char count;
     unsigned char var;
     
-    static const int numSamples=64; //Number of samples for variance compuation
+    static const int numSamples=32; //Number of samples for variance compuation
     static const int fp=64; //Fixed point, log2(fp) bits are the decimal part
 };
 
diff -ruN flopsync/flopsync_dynamic_comb.cpp flopsync_temp/flopsync_dynamic_comb.cpp
--- flopsync/flopsync_dynamic_comb.cpp	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/flopsync_dynamic_comb.cpp	2013-07-28 22:54:50.000000000 +0200
@@ -59,6 +59,7 @@
     {
         if(flooder.synchronize()) flooder.resynchronize();
         
+#if 0
         unsigned int frameStart=flooder.getFrameStart();
         unsigned int start;
         //Node1 gets even comb slots, node2 even slots
@@ -88,5 +89,6 @@
             nrf.endWritePacket();
             nrf.setMode(Nrf24l01::SLEEP);
         }
+#endif
     }
 }
diff -ruN flopsync/flopsync_rootnode_comb.cpp flopsync_temp/flopsync_rootnode_comb.cpp
--- flopsync/flopsync_rootnode_comb.cpp	2013-07-29 00:02:51.791594009 +0200
+++ flopsync_temp/flopsync_rootnode_comb.cpp	2013-07-28 23:05:25.000000000 +0200
@@ -33,8 +33,16 @@
 
 using namespace std;
 
+#include <miosix.h>
+//--------------
+typedef miosix::Gpio<GPIOC_BASE,8> fixme;
+//--------------
+
 int main()
 {
+    fixme::mode(miosix::Mode::OUTPUT);
+    fixme::low();
+    
     puts(experimentName);
     const unsigned char address[]={0xab, 0xcd, 0xef};
     Nrf24l01& nrf=Nrf24l01::instance();
@@ -49,6 +57,7 @@
     {
         flooder.synchronize();
         
+#if 0
         puts("----");
         unsigned int frameStart=flooder.getFrameStart();
         for(unsigned int i=combSpacing,j=0;i<nominalPeriod-combSpacing/2;i+=combSpacing,j++)
@@ -83,5 +92,6 @@
                 printf("node%d.e=%d\n",(j & 1)+1,measuredTime-(frameStart+i));
             }
         }
+#endif
     }
 }
